_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -169,6 +180,25 @@ This `summary` element is [focusable][].
 </details>
 ```
 
+#### Failed Example 7
+
+This `a` element is [focusable][] because it fails to move focus when it receives focus. This is in contrast to Passed Example 4.
```

Comment:
We usually do not refer from one example to an other.
Notably because that would easily become a nightmare to maintain if some example number changes and we have to remember another example description 😖

Reply:
Will change to "This is in contrast to a focus sentinel that immediately jumps focus to a valid location." Is that better?